data/json/bionics.json

```diff
@@ -613,31 +613,31 @@
     "id": "bio_metabolics",
     "type": "bionic",
     "name": "Metabolic Interchange",
-    "description": "Your digestive system and power supply are interconnected.  Bionic energy is replenished at the cost of increasing hunger.  Can be toggled on and off at will.",
+    "description": "Your digestive system and power supply are interconnected.  Bionic energy is replenished at the cost of decreasing nutrition.  Can be toggled on and off at will.",
     "occupied_bodyparts": [ [ "TORSO", 25 ] ],
     "flags": [ "BIONIC_POWER_SOURCE", "BIONIC_TOGGLED", "BIONIC_NPC_USABLE" ]
   },
   {
     "id": "bio_meteorologist",
     "type": "bionic",
     "name": "Weather Reader",
-    "description": "A multitude of scientific instruments and sensors collect environmental data.  The data is compiled and presented as a simple readout of the current weather.  It also passively tells you your external temperature.",
+    "description": "A multitude of scientific instruments and sensors collect environmental data.  The data is compiled and presented as a simple readout of the current weather.  It also passively tells you the local air temperature.",
     "occupied_bodyparts": [ [ "TORSO", 1 ], [ "HEAD", 1 ] ],
     "act_cost": 10
   },
   {
     "id": "bio_nanobots",
     "type": "bionic",
     "name": "Repair Nanobots",
-    "description": "Inside your body is a fleet of tiny dormant robots.  While activated they will flit about your body, repairing damage and stopping bleeding at the cost of power.",
+    "description": "Inside your body is a fleet of tiny dormant robots.  While activated they will flit about your body, rapidly repairing damage and stopping bleeding at the cost of power.",
```

Comment:
I'm not sure `rapidly` adds much information, maybe give the actual healing rate?

Reply:
That would be better since all other CBMs also give exact values.